docs/tutorial/gatsby-image-tutorial

```diff
@@ -0,0 +1,275 @@
+---
+title: "Using Gatsby-Image With Your Site Tutorial"
+---
```

Comment:
To follow the template in [docs-templates](https://www.gatsbyjs.org/contributing/docs-templates/#tutorials) we should do a little more setup:

```
## What’s contained in this tutorial?

By the end of this tutorial, you’ll have done the following:

- learned how to use `gatsby-image` for responsive images
- queried for a single image with GraphQL
- sourced multiple images through YAML files
- learned how to troubleshoot common errors
```